tests/test_urls.py

```diff
@@ -119,15 +121,17 @@ def test_missing_parameters_for_new_issue(self):
 
     def test_new_issue_should_not_crash(self):
         '''Checks 500 is not accepted for /issues/new POST.'''
-        data = {'problem_category': u'mobile_site_bug',
-                'username': u'',
-                'description': u'foo',
-                'submit-type': u'github-proxy-report',
-                'url': u'http://example.com',
-                'os': u'Foobar',
-                'browser': u'BarFoo'}
-        rv = self.app.post('/issues/new', data=data)
-        self.assertNotEqual(rv.status_code, 500)
+        with patch('requests.post') as github_data:
+            github_data.status_code.return_value = 500
+            data = {'problem_category': u'mobile_site_bug',
+                    'username': u'',
+                    'description': u'foo',
+                    'submit-type': u'github-proxy-report',
+                    'url': u'http://example.com',
+                    'os': u'Foobar',
+                    'browser': u'BarFoo'}
+            rv = self.app.post('/issues/new', data=data)
+            self.assertNotEqual(rv.status_code, 500)
```

Comment:
so indeed. The test is working if we cut the network, but the test is also posting an issue with the network. So as @brizental said, that smell false positive.

+1 to `self.assertEqual(rv.status_code, 400)`

and indeed we need to figure out why it is still trying to post the issue like in 
https://github.com/webcompat/webcompat-tests/issues/1239